lib/turbo_tests/runner.rb

```diff
@@ -96,7 +101,11 @@ def run
 
       @threads.each(&:join)
 
-      @reporter.failed_examples.empty? && wait_threads.map(&:value).all?(&:success?)
+      if @reporter.failed_examples.empty? && wait_threads.map(&:value).all?(&:success?)
+        SUCCESS_EXIT_CODE
+      else
+        wait_threads.max { |value| value.exitstatus }.value.exitstatus
```

Comment:
Hand back the highest exit code for failures -- same as parallel-tests: https://github.com/grosser/parallel_tests/blob/01218db8d43fab939fe5b53738af1fab771373dd/lib/parallel_tests/cli.rb#L90

Reply:
It fails with `undefined method 'exitstatus' for #<Process::Waiter:0x0000000002588318 dead> (NoMethodError)` when I test it on a big test suite.

```
Finished in 4 minutes 38.4 seconds (files took 26.8 seconds to load)
16001 examples, 11 failures, 1 pending, 2 errors occurred outside of examples

# ...

bundler: failed to load command: turbo_tests (/home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bin/turbo_tests)
Traceback (most recent call last):
        22: from /home/serpapi/.rbenv/versions/2.7.2/bin/bundle:25:in `<main>'
        21: from /home/serpapi/.rbenv/versions/2.7.2/bin/bundle:25:in `load'
        20: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/exe/bundle:37:in `<top (required)>'
        19: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
        18: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/exe/bundle:49:in `block in <top (required)>'
        17: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/cli.rb:25:in `start'
        16: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
        15: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/cli.rb:31:in `dispatch'
        14: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
        13: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
        12: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
        11: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/cli.rb:477:in `exec'
        10: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/cli/exec.rb:23:in `run'
         9: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/cli/exec.rb:58:in `kernel_load'
         8: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/bundler-2.2.27/lib/bundler/cli/exec.rb:58:in `load'
         7: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bin/turbo_tests:25:in `<top (required)>'
         6: from /home/serpapi/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/bin/turbo_tests:25:in `load'
         5: from /home/serpapi/Workspace/turbo_tests/bin/turbo_tests:11:in `<top (required)>'
         4: from /home/serpapi/Workspace/turbo_tests/lib/turbo_tests/cli.rb:101:in `run'
         3: from /home/serpapi/Workspace/turbo_tests/lib/turbo_tests/runner.rb:41:in `run'
         2: from /home/serpapi/Workspace/turbo_tests/lib/turbo_tests/runner.rb:107:in `run'
         1: from /home/serpapi/Workspace/turbo_tests/lib/turbo_tests/runner.rb:107:in `max'
/home/serpapi/Workspace/turbo_tests/lib/turbo_tests/runner.rb:107:in `block in run': undefined method `exitstatus' for #<Process::Waiter:0x0000000002588318 dead> (NoMethodError)
```